apps/web/src/components/Dropdown/Dropdown.tsx

```diff
@@ -71,13 +77,15 @@ export const Dropdown = ({
       return
     }
 
-    if (buttonRef.current?.getAttribute('aria-expanded') !== 'true') {
-      return
-    }
-
-    buttonRef.current.click()
+    setIsOpen(false)
   })
 
+  useEffect(() => {
+    if (buttonRef.current) {
+      buttonRef.current.setAttribute('aria-expanded', isOpen ? 'true' : 'false')
```

Comment:
Totalement facultatif :
```suggestion
      buttonRef.current.setAttribute('aria-expanded', `${isOpen}`)
```